compiler/src/main/java/org/qbicc/graph/SimpleBasicBlockBuilder.java

```diff
@@ -527,7 +532,7 @@ public Value classOf(Value typeId) {
         ValueType idType = typeIdType.getUpperBound();
         if (idType instanceof ReferenceType) {
             CompilationContext ctxt = classContext.getCompilationContext();
-            ctxt.warning(getLocation(), "TODO: class type ID argument is given as a reference type which is not allowed");
+//            ctxt.warning(getLocation(), "TODO: class type ID argument is given as a reference type which is not allowed");
```

Comment:
I know this produces a ton of output, but did you intend to comment out the warning?  If so, maybe leave a TODO or open an issue so we don't lose track of it?

Reply:
Yeah it's just way too noisy.  I'll open a follow-up issue.

Reply:
Done (#570).